tests/main/driver/test_restart_serial.py

```diff
@@ -122,6 +122,9 @@ def test_restart_save_to_disk():
             if isinstance(
                 driver_state.physics_state.__dict__[var],
                 np.ndarray,
+            ) or isinstance(
+                driver_state.physics_state.__dict__[var],
+                Quantity,
             ):
```

Comment:
Did you know that you can just give a tuple with all valid classes as second argument to `isinstance`?

```suggestion
            if isinstance(
                driver_state.physics_state.__dict__[var],
                (np.ndarray, Quantity)
            ):
```

Reply:
I actually did not, so that's a great suggestion!